services-geojson/src/main/java/com/mapbox/geojson/Feature.java

```diff
@@ -48,7 +49,7 @@
  * @since 1.0.0
  */
 @Keep
-public final class Feature implements GeoJson {
+public final class Feature implements GeoJson, Serializable {
```

Comment:
What's the advantage of making `Feature`/`FeatureCollection` serializable now? If we'd like to make them serializable in general, why not make `GeoJson` implement `Serializable`?

Reply:
I noticed inconsistency that felt odd, so decided to add it there. I think you are right: we can just add Serializable to GeoJson. Will experiment

Reply:
I decide to remove these changes from this PR.
I think your suggestion is a valid one and should be addressed in a separate PR.